docs/source/author/topics.rst

```diff
@@ -613,11 +613,14 @@ Curriculum Areas Configuration File
 
       - ``name:`` The name of the curriculum area (this is what will be displayed to the
         user).
+      - ``colour:`` The CSS colour class to use for colouring the curriculum
+        area badge on the website.
+        This colour is also applied to all children of curriculum area.
```

Comment:
I think it should just be added to the documentation, just a note somewhere listing all the available colours so that people don't have to go digging through the source code.